### PR TITLE
Feat dynamic header

### PR DESCRIPTION
**Pull request by ImieboGodson**
- Added dynamic header based on if user is signed in or not
- Fixed conditional statement in App.js
- Route are in small letters

### CODE REVIEW

File: frontend/src/components/Navbar.jsx [hunk 1]
```diff
@@ -1,164 +1,65 @@
...
+                  <Link to="/" className="hover:text-[#41DC65] text-[#05377F] px-3 py-2 rounded-md text-xs tracking-tight leading-3 font-bold flex justify-center items-center"><BiWalletAlt className='mr-[1px]' size='1.2em'/> Wallet</Link>
+                )
+                :
+                (
+                  ''
+                )
+              }
+              <Link to="/" className="hover:text-[#41DC65] text-[#05377F] px-3 py-2 rounded-md text-xs tracking-tight leading-3 font-bold flex justify-center items-center"><MdOutlinePersonOutline className='mr-[1px]' size='1.2em'/> About Us</Link>
+              <Link to="/faqs" className="hover:text-[#41DC65] text-[#05377F] px-3 py-2 rounded-md text-xs tracking-tight leading-3 font-bold">FAQs</Link>
+              <Link to="/" className="hover:text-[#41DC65] text-[#05377F] px-3 py-2 rounded-md text-xs tracking-tight leading-3 font-bold">Support</Link>
```
Comment (MorganJay) [inline frontend/src/components/Navbar.jsx]: Isn't line 41 a link to the support route?

File: frontend/src/components/Navbar.jsx [hunk 2]
```diff
@@ -1,164 +1,65 @@
...
-          <div className="-mr-2 flex md:hidden">
-            <button
-              onClick={() => setIsOpen(!isOpen)}
-              type="button"
-              className="bg-gray-800 inline-flex items-center justify-center p-2 rounded-md text-matic hover:text-white hover:bg-gray-700 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-offset-gray-800 focus:ring-white"
-              aria-controls="mobile-menu"
-              aria-expanded="false"
-            >
-              <span className="sr-only">Open main menu</span>
-              {!isOpen ? (
```
Comment (MorganJay) [inline frontend/src/components/Navbar.jsx]: All these Transition and open and close menu aren't needed anymore?

File: frontend/src/components/Navbar.jsx (hunk 1 above)
Comment (ImieboGodson) [reply frontend/src/components/Navbar.jsx]: That route is yet to be created.

File: frontend/src/components/Navbar.jsx (hunk 2 above)
Comment (ImieboGodson) [reply frontend/src/components/Navbar.jsx]: will sort that

File: frontend/src/components/Navbar.jsx (hunk 1 above)
Comment (MorganJay) [reply frontend/src/components/Navbar.jsx]: So why do you have it in that function to render the footer? 

You can just add the /support here tho👍

**Comment by ImieboGodson** [comment]
@MorganJay kindly check for requested changes

File: frontend/src/components/Navbar.jsx [hunk 3]
```diff
@@ -116,46 +121,41 @@ const Navbar = () => {
...
-                </a>
-                <a
-                  href="/"
-                  className=" hover:bg-gray-700 hover:text-white block px-3 py-2 rounded-md text-base font-medium"
-                >
-                  Login
-                </a>
+                <Link to="/" className="hover:text-[#41DC65] text-[#05377F] px-3 py-2 rounded-md text-xs tracking-tight leading-3 font-bold flex justify-center items-center"><MdOutlineHome className='mr-[1px]' size='1.2em'/> Home</Link>
+                <Link to="/" className="hover:text-[#41DC65] text-[#05377F] px-3 py-2 rounded-md text-xs tracking-tight leading-3 font-bold flex justify-center items-center"><RiBankLine className='mr-[2px]' size='1.2em'/> Transfer</Link>
+                {
```
Comment (MorganJay) [inline frontend/src/components/Navbar.jsx]: Ever heard of DRY? 

These Links are repeated in the normal nav and mobile nav

**Review by MorganJay — APPROVED** [review]

**Pull request merged**